### PR TITLE
Add: Loading icons and skeletons for D/C, tables, and graph

### PR DESCRIPTION
**Pull request by BruceJi7**

### CODE REVIEW

File: src/ui/Skeletos/DuplicateSkeleton/DuplicateSkeleton.tsx [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import { ReactElement } from "react"
+
+type Props = {
+  howMany: number
+  children: ReactElement
+}
+
+export const DuplicateSkeleton = (props: Props) => {
```
Comment (ivanms1) [inline src/ui/Skeletos/DuplicateSkeleton/DuplicateSkeleton.tsx:8]:
```suggestion
export const DuplicateSkeleton = ({ howMany, children }: Props) => {
```

File: src/containers/TxnActivitySection/TxnActivitySection.tsx [hunk 2]
```diff
@@ -124,9 +127,19 @@ export const TxnActivitySection = (props: Props) => {
 
         {/* TXN Activity - Table for each block made - shows hashes, created at, etc*/}
 
-        {txnActivityData.map((i) => (
+        {/* {txnActivityData.map((i) => (
           <TxnDataBlock key={i.txhash} fullPage={fullPage} data={{ ...i }} />
```
Comment (ivanms1) [inline src/containers/TxnActivitySection/TxnActivitySection.tsx:131]: remove please

File: src/ui/Skeletos/DuplicateSkeleton/DuplicateSkeleton.tsx (hunk 1 above)
Comment (BruceJi7) [reply src/ui/Skeletos/DuplicateSkeleton/DuplicateSkeleton.tsx:8]: Yes, that might be a good idea. I should go through and switch over to destructured at some point

File: src/ui/Skeletos/MainTableSkeleton/MainTableSkeleton.css [hunk 3]
```diff
@@ -0,0 +1,29 @@
+.skeleton-table {
+  /* padding: 10px; */
```
Comment (ivanms1) [inline src/ui/Skeletos/MainTableSkeleton/MainTableSkeleton.css:2]: remove please

**Review by ivanms1 — APPROVED** [review]

**Pull request merged**